STT/Extensions/DateExtensions.swift

```diff
@@ -161,4 +172,16 @@ public extension Date {
         
         return gregorianCalendar.date(from: dateComponents)
     }
+    
+    ///
+    /// Detect if time format is in 12hr or 24hr format
+    ///
+    func is12hClockFormat() -> Bool {
+        let formatString = DateFormatter.dateFormat(
+            fromTemplate: "j",
+            options: 0,
+            locale: Locale.current
+        )!
```

Comment:
Force unwrap 😢

Reply:
I believe in this case it's acceptable as we have definite and static data